index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div>
+                <a class="mylogo" href="#top">My Logo</a>
+            </div>
+            <nav class="menuholder">
+                <img src="src/menuicon.png" alt="test">
+    
+            </nav>
+        </div>
+    </header>
+    
+    <section class="container2">
+        <div class="headerholder headmargin"> 
+            <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
```

Comment:
- Kindly update the name used in the design to your name as this would be your personal portfolio you'd be building gradually from scratch to finish.